app/javascript/hooks/mutations/users/useDeleteUser.jsx

```diff
@@ -0,0 +1,23 @@
+import { useMutation, useQueryClient } from 'react-query';
+import axios from 'axios';
```

Comment:
You can use the custom axios and the `ENDPOINTS` constant instance to further simplify this.

```
import axios, { ENDPOINTS } from '../../../helpers/Axios';
```

Reply:
I thought we settled on not using the ENDPOINTS for now

Reply:
We haven't discussed its use for now.
I'm still using it and I think Hadi used it in his last PR.
We should discuss it later on to keep consistency.